infra/cifuzz/fuzz_target.py

```diff
@@ -254,6 +255,15 @@ def is_crash_reportable(self, testcase):
       logging.info('Failed to reproduce the crash using the obtained testcase.')
       return False
 
+    is_novel = self.is_crash_novel(testcase)
+
+    if is_novel:
+      return True
+    return False
+
+  def is_crash_novel(self, testcase):
+    """Returns False if the crash couldn't be reproduced on the ClusterFuzz
```

Comment:
Nit: docstring is confusing with the inversions (and seems wrong?).

Reword as something like """Returns whether or not the crash is new. A crash is considered new if it can't be reproduced on an older ClusterFuzz build of the target."""

Reply:
Ah sorry about this mistake. I used your docstring. Thanks!